now.json

```diff
@@ -2,15 +2,14 @@
   "name": "carbon",
   "version": 2,
   "public": true,
-  "builds": [{ "src": "package.json", "use": "@now/static-build" }],
+  "builds": [{ "src": "package.json", "use": "@now/next@canary" }],
   "routes": [
     { "src": "^/about(.*)", "dest": "/about" },
     { "src": "^/embed(.*)", "dest": "/embed" },
     { "src": "^/_next/(.*)", "dest": "/_next/$1" },
```

Comment:
@timneutkens can I remove this line again now? Or is it still required?

Reply:
Done!